internal/runner/runner.go

```diff
@@ -65,6 +65,23 @@ func (*Default) CmdCombinedOutput(cmd *exec.Cmd) ([]byte, error) {
 	return cmd.CombinedOutput()
 }
 
+// Dry is the Runner that only prints commands and does not execute them.
+type Dry struct{}
+
+func (*Dry) CmdRun(cmd *exec.Cmd) error {
+	log.Printf("%v", cmd.String())
```

Comment:
Should this be fmt.Println?

Reply:
I tried it. I don't think it looks much nicer; it's at a different indentation from, and interleaved with, the log messages, so it's hard to scan. Example:
```
2020/05/11 11:52:31 Running: [terraform show -json /tmp/699780966]
2020/05/11 11:52:32 Found importable resource: "module.network.module.subnets.google_compute_subnetwork.subnetwork[\"us-central1/forseti-subnet\"]"
/usr/local/bin/terraform import module.network.module.subnets.google_compute_subnetwork.subnetwork["us-central1/forseti-subnet"] projects/mpetkov-hcls-testing-monitor/regions/us-central1/subnetworks/forseti-subnet
2020/05/11 11:52:32 Found importable resource: "module.network.module.vpc.google_compute_network.network"
/usr/local/bin/terraform import module.network.module.vpc.google_compute_network.network projects/mpetkov-hcls-testing-monitor/global/networks/forseti-vpc
2020/05/11 11:52:32 Found importable resource: "module.router.google_compute_router.router"
/usr/local/bin/terraform import module.router.google_compute_router.router projects/mpetkov-hcls-testing-monitor/regions/us-central1/routers/forseti-router
2020/05/11 11:52:32 Found importable resource: "module.router.google_compute_router_nat.nats[\"forseti-nat\"]"
/usr/local/bin/terraform import module.router.google_compute_router_nat.nats["forseti-nat"] projects/mpetkov-hcls-testing-monitor/regions/us-central1/routers/forseti-router/forseti-nat
2020/05/11 11:52:32 No resources imported.
```

Alternative would be to have the runner return the command as output, and have the importer collect them and print them all at once at the end.